src/styles/overflowXHidden/types.ts

```diff
@@ -1,5 +1,5 @@
-import {CSSProperties} from 'react';
+import {ViewStyle} from 'react-native';
 
-type OverflowXHiddenStyles = Partial<Pick<CSSProperties, 'overflowX'>>;
+type OverflowXHiddenStyles = Pick<ViewStyle, 'overflowX'>;
```

Comment:
cant' we just do `ViewStyle['overflowX']` when we use this type?

Reply:
Actually this is totally different type when using `ViewStyle['overflowX']` its resulting that `OverflowXHiddenStyles` will expect type `Property.OverflowX | undefined` but by using `Pick<ViewStyle, 'overflowX'>` its resulting in type `{
    overflowX?: Property.OverflowX | undefined;
}` which is correct type

Reply:
ah I see. I think I got confused because I didn't know the usage of `OverflowXHiddenStyle`

Do we actually have to create a separate type for a style object that only contains `overflowX`? Can't we use `ViewStyle` in `styles/overflowXHidden/index.ts` and `styles/overFlowXHidden/index.native.ts`?

<img width="500" alt="Screenshot 2023-10-10 at 4 13 47 PM" src="https://github.com/Expensify/App/assets/98560306/d2cf30b5-db0f-4db6-b224-c92672dac7c3">

Reply:
@hayata-suenaga We cannot do that for example https://github.com/Expensify/App/blob/a63f853381cd6ea680c31ee24c7c72f02e2d1513/src/styles/styles.js#L1716-L1727
we are spreding `overflowXHidden` if it was typed as ViewStyle it will result in TS error because it will think in overflowXHidden there are properties which are already defined